src/lib/Client.ts

```diff
@@ -233,11 +239,20 @@ export class Client extends EventEmitter {
 
         // #region Keep alive
         // Send a KeepAlive packet every second
+        let lastKeepAlive = new Date().getTime();
```

Comment:
I don't think this keep-alive check is functional, there's no condition that won't cause it to be updated every interval

Reply:
![image](https://user-images.githubusercontent.com/1159091/198858316-778ee5f6-cc37-4cd5-b0dc-0eadf5fe70ea.png)

Reply:
I see your point. But at some point in testing, I got the error that when my computer (Windows) got to sleep, the connection was still not "disconnected", but the mixer did close the connection because no keep-alive got send in that time.
This problem is handled here, if the program is haltet by the operating system and is resumed later on, the time stamps differ. I should probably add a comment for this in the code...

Reply:
Right, got it!